python/src/aac/lang/active_context_lifecycle_manager.py

```diff
@@ -21,7 +21,7 @@ def get_active_context(reload_context: bool = False) -> LanguageContext:
     """
     global ACTIVE_CONTEXT
 
-    if not ACTIVE_CONTEXT and not reload_context:
+    if not ACTIVE_CONTEXT and not reload_context and False:  # Temporary disabling until we get state management plugins.
```

Comment:
If we're disabling it anyways, we may as well short-circuit a bit sooner. 🙃 

```suggestion
    if False and not ACTIVE_CONTEXT and not reload_context:  # Temporary disabling until we get state management plugins.
```

Reply:
🤷‍♂️ It's just short-circuiting two other bool comparisons, but sure I can move it up.